### ISSUE TITLE
Need option to include the current configuration name as part of the filename

### ISSUE THREAD

**Issue by rbrian** [WilkoV/Fusion360_ExportIt#58]
Title says it all. But for an example, if I have a design called "Foo Object", and configurations "1.x", "2.x", and "3.x", it'd be nice to have the either have the output folder include the "1.x", or maybe the filename inside the folder have it.

**Comment by WilkoV** [comment]
I'm not  a 100% sure what configuration is in this context.  Are you talking about [this](https://help.autodesk.com/view/fusion360/ENU/?guid=CFG-OVERVIEW)? If yes, I'm not able to help, because I don't have this subscription.

**Comment by rbrian** [comment]
yes, that's the thing. guess i'll look at the code and see if i can figure out where to add it. i remember seeing the API calls to get the current configuration name, but that's about it. will do a PR if I get it working.

**Comment by WilkoV** [comment]
This would be great. PR are very welcome.

**Comment by rbrian** [comment]
Just did the pull request. Works beautifully as far as I can tell!
